src/components/Toggle/Toggle.tsx

```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
               role='button'
               aria-pressed={ isChecked }
               onClick={ this._onClick }
+              aria-describedby={ this._id + 'ariaDescribedBy' }
             />
             <div className='ms-Toggle-background'>
               <div className='ms-Toggle-focus' />
               <div className='ms-Toggle-thumb' />
             </div>
-            <Label className='ms-Toggle-stateText'>{ stateText }</Label>
+            <Label
+              className='ms-Toggle-stateText'
+              role='status'
+              aria-live='polite'
```

Comment:
This is overkill and shouldn't be needed. There's only 2 states. The current state gets read, and activating the button flips it.